cohort_4/playbook/activities.md

```diff
@@ -24,6 +29,59 @@
 
 TODO
 
+## Brainstorming
+
+### Forming groups
+
+- Assign people to groups.
+  - There are social pressures between friends when forming groups. Start by
+    assigning the hackers to random groups.
+  - If any problems arise in a group handle it on a case-by-case basis.
+
+### Yes-and sessions
+
+- "Yes-and" sessions are exercises that help people get out of a judgmental
+  mindset before brainstorming
+  - In a "Yes-and" session, pairs of hackers build off each-others' ideas to
+    create long, completely nonsensical ideas
+- To run a "Yes-and" session, start with an example between facilitators
+  - ex: https://youtu.be/Qe2a3ppacUk
+  - ex: between person A and person B:
+
+    ```
+    > A: Let's create tinder for unicorns
+    > B: And it will be super effective, and the unicorn population will increase 1,000 times
+    > A: Yes, and there won't be enough food for the unicorns
+    > B: So they'll build a spaceship to search for new planets to colonize!
+    ```
```

Comment:
Any reason the blockquotes are within a fenced code block?

Example of blockquotes:

```
> this is a rendered
> blockquote!
```

Rendered:

> this is a rendered
> blockquote!